extension/chrome/elements/compose-modules/compose-draft-module.ts

```diff
@@ -125,9 +128,7 @@ export class ComposeDraftModule extends ViewModule<ComposeView> {
         if (!this.view.draftId || this.isLocalDraftId(this.view.draftId)) {
           const draftId = await this.doUploadDraftWithLocalStorageFallback(mimeMsg, msgData, async () => {
             const { id } = await this.view.emailProvider.draftCreate(mimeMsg, this.view.threadId);
-            if (this.isLocalDraftId(this.view.draftId)) { // delete local draft if there is one
-              await storageLocalRemove([this.view.draftId]);
-            }
+            await this.localDraftRemove(); // delete local draft if there is one
```

Comment:
have you considered moving this call to doUploadDraftWithLocalStorageFallback()?

Reply:
Done 👍